docs/user-guide/troubleshooting.md

```diff
@@ -8,16 +8,54 @@ Do not move it without providing redirects.
+on the lower left hand side.
+```
+
+ Workstation                                        Kubernetes
++------------------+                               +----------------------------------------------------+
+|                  |                               |                                                    |
+|                  |                               |                                                    |
+|  +------------+  |                               |  +------------+   apiserver        +------------+  |
+|  |            |  |      user authentication      |  |            |   proxy            |            |  |
+|  |            +------------------------------------>+            +------------------->+            |  |
+|  |            |  |                               |  |            |                    |            |  |
+|  +--------+---+  |                               |  |            |                    |            |  |
+|  kubectl  ^      |                               |  |            |                    |            |  |
+|  proxy    |      |                          +------>+            |   service          |            |  |
+|           |      |                          |    |  |            |   authentication   |            |  |
+|           |      |                          |    |  |            +<-------------------+            |  |
+|  +--------+---+  |                          |    |  |            |                    |            |  |
+|  |            |  |                          |    |  +------------+                    +------------+  |
+|  |            +-----------------------------+    |    apiserver                         dashboard     |
+|  |            |  |      user authentication      |                                                    |
+|  +------------+  |      with username &          |                                                    |
+|  browser         |      password only            |                                                    |
+|                  |                               |                                                    |
+|                  |                               |                                                    |
++------------------+                               +----------------------------------------------------+
+
+```
+__User authentication__
+
+Browsers can only authenticate with username and password. If youre apiserver is setup accordingly,
+ then you can access directly with `https://<master>/ui`. Otherwise, you can start a local proxy
+ with `kubectl proxy` and access Dashboard with the URL `http://localhost:8001`.
+
+__Service authentication__
+
+If you have successfully authenticated, then apiserver proxy will forward the request to dashboard backend. However, the dashboard backend will need information from apiserver and has to authenticate with apiserver again.
```

Comment:
...successfully **passed authentication**, then... 
...request to **the** Dashboard **backend**. ... 
... and **will have** to  authenticate with apiserver...